### PR TITLE
removed default_value and variant in Proto definition

### PR DESCRIPTION
**Pull request by James-Milligan**
Removal of default_value from request:
https://github.com/open-feature/flagd/issues/69

Removal of variant from response:
Not present in README.md expected responses, likely an artefact from previous OpenAPI definition
https://github.com/open-feature/flagd/blob/main/README.md

Signed-off-by: James-Milligan <james@omnant.co.uk>

### CODE REVIEW

File: protobuf/v1/schema.proto [hunk 1]
```diff
@@ -10,70 +10,62 @@ import "google/api/annotations.proto";
 message ResolveBooleanRequest {
     string flag_key = 1;
     google.protobuf.Struct context = 2;
-    bool default_value = 3;
 }
 message ResolveBooleanResponse {
     bool value = 1;
     string reason = 2;
-    string variant = 3;
```
Comment (beeme1mr) [inline protobuf/v1/schema.proto:18]: Why did you remove the variant?

File: protobuf/v1/schema.proto [hunk 2]
```diff
@@ -10,70 +10,62 @@ import "google/api/annotations.proto";
...
 
 message ResolveStringRequest {
     string flag_key = 1;
     google.protobuf.Struct context = 2;
-    string default_value = 3;
 }
 message ResolveStringResponse {
     string value = 1;
     string reason = 2;
-    string variant = 3;
```
Comment (beeme1mr) [inline protobuf/v1/schema.proto:29]: Same question as above.

File: protobuf/v1/schema.proto [hunk 3]
```diff
@@ -10,70 +10,62 @@ import "google/api/annotations.proto";
...
 
 message ResolveNumberRequest {
     string flag_key = 1;
     google.protobuf.Struct context = 2;
-    float default_value = 3;
 }
 message ResolveNumberResponse {
     float value = 1;
     string reason = 2;
-    string variant = 3;
```
Comment (beeme1mr) [inline protobuf/v1/schema.proto:40]: Same question as above.

**Review by beeme1mr — CHANGES_REQUESTED** [review]
It looks like this PR removed the variant from the response. Was this intended?

File: protobuf/v1/schema.proto (hunk 1 above)
Comment (James-Milligan) [reply protobuf/v1/schema.proto:18]: Variant was never populated by the IEvaluator interface, JSONEvaluator struct or HTTPService. It was also never referenced in the README for any expected responses.
It looks like an artefact from the original proto generation from the existing OpenAPI spec using gnostic as there is reference to it in providers.yaml but is never populated.
